Week 3/Homework/iceCream.rb

```diff
@@ -0,0 +1,108 @@
+brands = {
+	"Noah's Ice Cream" => {
+		"owner" => "Noah",
+		"location" => "NYC",
+		"website_url" => "http://www.noahs-icecream.herokuapp.com"
+	},
+	"Friendly's Ice Cream"  => {
+		"owner" => "John M. Maguire",
+		"location" => "Wilbraham, MA",
+		"website_url" => "https://www.friendlys.com/ice-cream/"
+	},
+	"Baskin Robbins"  => {
+		"owner" => "Burt Baskin and Irv Robbins",
+		"location" => "Canton, MA",
+		"website_url" => "https://www.baskinrobbins.com/content/baskinrobbins/en.html"
+	}
+}
+
+ice_cream_flavors = {
+	"standard_flavors" => {
+		"chocolate" => {
+			"cost_in_cents_per_ounce" => 20,
+			"amount_in_stock_in_ounces" => 500000,
+			"brand" => brands["Noah's Ice Cream"]
```

Comment:
I like this setup, downside is that if you change `Noah's Ice Cream` shop to `Kiera's Ice Cream` what happens? This is actually where we would use `CONSTANTS`.